src/modules/rlm_couchbase/couchbase.h

```diff
@@ -26,11 +26,13 @@
  */
 RCSIDH(couchbase_h, "$Id$")
 
-DIAG_OFF(error)
+#ifdef HAVE_WDOCUMENTATION
```

Comment:
Do the same here as you did fo rlm_mruby.h and rlm_unbound.c.  It's not the header that's optionally included, just the macros.

Reply:
done